<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QTextEdit" name="Txt_Status">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;Status:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_copy">
       <attribute name="title">
        <string>Copy</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Source Server:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="Copy_list_Src"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Destination Server:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="Copy_list_Dest"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="Copy_Btn_ServerAdd">
                <property name="text">
                 <string>Add Server to List</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Copy_Line_ServerAdd"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Source PG Account:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="Copy_Line_SrcAcc">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Source PG Password</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="Copy_Line_SrcPass">
                    <property name="inputMethodHints">
                     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Destination PG Account:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="Copy_Line_DestAcc">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Destination PG Password</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="Copy_Line_DestPass">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Source Database Name</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="Copy_Line_SrcDB">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Destination Database Name</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="Copy_Line_DestDB">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QCheckBox" name="Copy_Check_DBSameName">
                <property name="text">
                 <string>Source DB Name = Destination DB Name</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="Copy_Btn_RunCopy">
            <property name="text">
             <string>Start Transfer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_backup">
       <attribute name="title">
        <string>Backup</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_41" stretch="2,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_42">
              <item>
               <widget class="QLabel" name="label_33">
                <property name="text">
                 <string>Source Server:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="Backup_list_Src"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="2,2">
                <item>
                 <widget class="QPushButton" name="Backup_Btn_ServerAdd">
                  <property name="text">
                   <string>Add Source Server</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Backup_Line_ServerAdd"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_27">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_43">
                  <item>
                   <widget class="QLabel" name="label_34">
                    <property name="text">
                     <string>Source PG Account:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="Backup_Line_SrcAcc">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_44">
                  <item>
                   <widget class="QLabel" name="label_35">
                    <property name="text">
                     <string>Source PG Password</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="Backup_Line_SrcPass">
                    <property name="inputMethodHints">
                     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_48" stretch="1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_30">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_49">
                <item>
                 <widget class="QLabel" name="label_39">
                  <property name="text">
                   <string>Source Database Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Backup_Line_SrcDB">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="Backup_Btn_RunBackup">
            <property name="text">
             <string>Start Backup</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_restore">
       <attribute name="title">
        <string>Restore</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <item>
           <widget class="QPushButton" name="Restore_Btn_File">
            <property name="text">
             <string>Select Backup File (.sql)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Restore_Line_File">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_48">
            <property name="text">
             <string>Destination Server:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="Restore_list_Dest"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_38" stretch="2,2">
            <item>
             <widget class="QPushButton" name="Restore_Btn_ServerAdd">
              <property name="text">
               <string>Add Destination Server</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Restore_Line_ServerAdd"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Destination PG Account:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Restore_Line_DestAcc">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Destination PG Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Restore_Line_DestPass">
                <property name="text">
                 <string/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Destination Database Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Restore_Line_DestDB">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="Restore_Btn_RunRestore">
            <property name="text">
             <string>Start Restore</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>Copy_list_Src</tabstop>
  <tabstop>Copy_list_Dest</tabstop>
  <tabstop>Copy_Btn_ServerAdd</tabstop>
  <tabstop>Copy_Line_ServerAdd</tabstop>
  <tabstop>Copy_Line_SrcAcc</tabstop>
  <tabstop>Copy_Line_SrcPass</tabstop>
  <tabstop>Copy_Line_DestAcc</tabstop>
  <tabstop>Copy_Line_DestPass</tabstop>
  <tabstop>Copy_Line_SrcDB</tabstop>
  <tabstop>Copy_Check_DBSameName</tabstop>
  <tabstop>Copy_Line_DestDB</tabstop>
  <tabstop>Copy_Btn_RunCopy</tabstop>
  <tabstop>Backup_list_Src</tabstop>
  <tabstop>Backup_Btn_ServerAdd</tabstop>
  <tabstop>Backup_Line_ServerAdd</tabstop>
  <tabstop>Backup_Line_SrcAcc</tabstop>
  <tabstop>Backup_Line_SrcPass</tabstop>
  <tabstop>Backup_Line_SrcDB</tabstop>
  <tabstop>Backup_Btn_RunBackup</tabstop>
  <tabstop>Restore_Btn_File</tabstop>
  <tabstop>Restore_Line_File</tabstop>
  <tabstop>Restore_list_Dest</tabstop>
  <tabstop>Restore_Btn_ServerAdd</tabstop>
  <tabstop>Restore_Line_ServerAdd</tabstop>
  <tabstop>Restore_Line_DestAcc</tabstop>
  <tabstop>Restore_Line_DestPass</tabstop>
  <tabstop>Restore_Line_DestDB</tabstop>
  <tabstop>Restore_Btn_RunRestore</tabstop>
  <tabstop>Txt_Status</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
